corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -147,6 +149,12 @@ hqDefine("geospatial/js/geospatial_map", [
                         self.handleDisbursementResults(ret['result']);
                     }
                 },
+                error: function () {
+                    alertUser.alert_user(
+                        gettext("Oops! Something went wrong! Please contact admin if the problem persists."), 'danger'
```

Comment:
Should we not rather link the user to creating a support ticket if the issue persists?

Reply:
That's a good idea